tutorials/R/download-explore/NEON-download-explore.Rmd

```diff
@@ -0,0 +1,379 @@
+---
+syncID: 5f9c4048a27749c19ee8ecfc78806363
+title: "Download and Explore NEON Data"
+description: "Tutorial for downloading data from the Data Portal and the neonUtilities package, then exploring and understanding the downloaded data"
+dateCreated:  2018-11-07
+authors: [Claire K. Lunch]
+contributors: [Christine Laney, Megan A. Jones]
+estimatedTime: 1 - 2 hours
+packagesLibraries: [devtools, geoNEON, neonUtilities, rhdf5, raster]
+topics: data-management, rep-sci
+languagesTool: R, API
+dataProduct:
+code1: R/download-explore/NEON-download-explore
+tutorialSeries: 
+urlTitle: neon-explore
+---
+
+
+This tutorial covers downloading NEON data, using the Data Portal and 
+the neonUtilities R package, as well as basic instruction in beginning to 
+explore and work with the downloaded data, including guidance in 
+navigating data documentation.
+
+## NEON data
+There are 3 basic categories of NEON data:
+
+1. Remote sensing (AOP) - Data collected by the airborne observation platform, 
+e.g. LIDAR, surface reflectance
+1. Observational (OS) - Data collected by a human in the field, or in an 
+analytical laboratory, e.g. beetle identification, foliar isotopes
+1. Instrumentation (IS) - Data collected by an automated, streaming sensor, e.g. 
+net radiation, soil carbon dioxide. This category also includes the eddy 
+covariance (EC) data, which are processed and structured in a unique way, distinct 
+from other instrumentation data.
+
+This lesson covers all four (sub)types of data. The download procedures are 
+similar for all types, but data navigation differs significantly by type.
+
+
+<div id="ds-objectives" markdown="1">
+
+## Objectives
+
+After completing this activity, you will be able to:
+
+* Download NEON data using the neonUtilities package.
+* Understand downloaded data sets and load them into R for analyses.
+
+## Things You’ll Need To Complete This Tutorial
+To complete this tutorial you will need the most current version of R and, 
+preferably, RStudio loaded on your computer.
+
+### Install R Packages
+
+* **devtools**: Needed to install packages from GitHub
+* **neonUtilities**: Basic functions for accessing NEON data
+* **raster**: Raster package; needed for remote sensing data
+* **geoNEON**: For working with NEON spatial data
+* **rhdf5**: HDF5 package; needed for eddy covariance data
+
+```{r packages, eval=FALSE}
+
+install.packages("devtools")
+install.packages("neonUtilities")
+install.packages("raster")
+devtools::install_github("NEONScience/NEON-geolocation/geoNEON")
+source("http://bioconductor.org/biocLite.R")
+biocLite("rhdf5")
+
+```
+
+
+### Additional Resources
+
+* <a href="https://www.neonscience.org/neonDataStackR" target="_blank">Tutorial for neonUtilities.</a> Some overlap with this tutorial but goes into more detail about the neonUtilities package.
```

Comment:
You may need to have a line of text before this.  I seem to remember that bullets don't render directly after a header on our website.